src/chrome/content.js

```diff
@@ -0,0 +1,108 @@
+$(function() {
+	let glossary = {}
+
+	$( document ).tooltip();
+
+	// scrapes language from the viva page
+	let language = 'english'
+	let langElement = $('.editorLoanLanguage')
+	if (langElement.length) {
+		let content = langElement.html()
+		language = content.substring(content.indexOf('<span>') + '<span>'.length, content.indexOf('<br>')).toLowerCase()
+	}
+
+	let glossaryName = language + '-glossary'
+
+	// get glossary from storage set by background.js
+	chrome.storage.local.get(glossaryName, (data) => {
+
+		let g = data[glossaryName]
+		glossary = {...g[''], ...g[getCountry()]}
+		console.log(`Finished loading ${language} glossary`)
+		getOriginalText()
+		highlightTerms(['.editorOriginalText:eq(1)', '.loan-description'])
+	})
+
+	// gives the popup the language and country of this page
+	chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+		    if (request.type == 'VIVA_PAGE_INFO') {
+		    	sendResponse({country: getCountry(), language: language})
+		    }
+	});
+
+	// scrapes the name of the country from the current Viva page
+	function getCountry() {
+		let location = $('a[href="#country-section"]').text()
+		let separator = location.lastIndexOf(',')
+		if (separator != -1) {
+			return location.slice(location.lastIndexOf(',') + 2)
+		} else {
+			return location.trim()
+		}
+		
+	}
+
+	// currentText stores the text without the tooltips so that the editor area doesn't have them
+	let currentText
+
+	function getOriginalText() {
+		currentText = $('.loan-description').text().trim()
+	}
+
+	$('.loan-description').click(function () {
+		$('.bodyText').val(currentText)
+	})
+
+	function saveCurrentText() {
+		currentText = $('.bodyText').val()
+	}
+
+	$('.editorSave:eq(2)').click(saveCurrentText)
+	$('.editorSave:eq(3)').click(saveCurrentText)
+	
+
+	// Below are my failed attempts to update the highlighted on change
```

Comment:
@kevinzwang meaning after the text is changed, going back and highlighting terms? Any thoughts on why it wasn't working?

Reply:
Yes, that is right. I believe it doesn't work because my listener is called before another one that's in the website's code that also changes the text, which overrides my changes.